web/src/site-admin/SiteAdminRepositoriesPermissionsPage.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react'
+import { RouteComponentProps } from 'react-router'
+import { Link } from 'react-router-dom'
+import { Subject, Observable } from 'rxjs'
+import { ActivationProps } from '../../../shared/src/components/activation/Activation'
+import { RepoLink } from '../../../shared/src/components/RepoLink'
+import * as GQL from '../../../shared/src/graphql/schema'
+import {
+    FilteredConnection,
+    FilteredConnectionQueryArgs,
+    FilteredConnectionFilter,
+} from '../components/FilteredConnection'
+import { PageTitle } from '../components/PageTitle'
+import { refreshSiteFlags } from '../site/backend'
+import { eventLogger } from '../tracking/eventLogger'
+import { fetchAllRepositoriesPermissions } from './backend'
+import { ErrorAlert } from '../components/alerts'
+import { Timestamp } from '../components/time/Timestamp'
+
+interface RepositoryPermissionsNodeProps extends ActivationProps {
+    node: GQL.IRepositoryPermissions
+    onDidUpdate?: () => void
+}
+
+interface RepositoryPermissionsNodeState {
+    errorDescription?: string
+}
+
+class RepositoryPermissionsNode extends React.PureComponent<
+    RepositoryPermissionsNodeProps,
+    RepositoryPermissionsNodeState
+> {
+    public state: RepositoryPermissionsNodeState = {}
+
+    public render(): JSX.Element | null {
+        return (
+            <li className="repository-node list-group-item py-2" data-e2e-repository={this.props.node.repository.name}>
```

Comment:
```suggestion
            <li className="repository-node list-group-item py-2">
```

seems like this is irrelevant unless you want to write e2e tests for this